src/mimeparser.rs

```diff
@@ -228,6 +232,7 @@ impl MimeMessage {
             group_avatar: None,
             failure_report: None,
             is_mime_modified: false,
+            decoded_data: mail_raw.clone(),
```

Comment:
maybe we can avoid this `clone()` here somehow (mail_raw is owned by mail that is borrowed to parser.parse_mime_recursive() some lines below), however, some things i've tried fail - if someone sees an easy way, hints would be welcome :) (probably worth some effort as this is this whole, decoded messages that may be easily several 10mb)

Reply:
i found a reasonable way myself, https://github.com/deltachat/deltachat-core-rust/pull/2151/commits/225ffd1dcc5c23a8b2d12cf0503e2334ccc23472